### ISSUE TITLE
Login and create account options have no focus styles

### ISSUE THREAD

**Issue by adamwoodnz** [Automattic/wp-calypso#99615]
These buttons still don't have focus styles:

![Screenshot 2025-02-11 at 09 58 43](https://github.com/user-attachments/assets/3d39e20c-2047-4ba1-9b8b-c3a02a0ed6c3)

_Originally posted by @jasmussen in https://github.com/Automattic/wp-calypso/pull/99314#pullrequestreview-2608093762_

Neither do these:

![Image](https://github.com/user-attachments/assets/6ba1f704-5efa-477a-9419-073528c5962c)

**Comment by jordesign** [comment]
Confirmed this in Triage. Nice catch @adamwoodnz

**Comment by jasmussen** [comment]
Noting that as I previously dove in with the hope of fixing this, I discovered that the first 3 buttons are Calypso components, but the "Continue with Email" is a core component. There's local CSS to override across, but it seems sensible, as part of this work, to standardize on one type of component.

**Comment by adamwoodnz** [comment]
Social button states shipped in https://github.com/Automattic/wp-calypso/pull/100807, but primary buttons remain highly inconsistent. Need another PR for that.
